docs/src/docs/asciidoc/user/writers.adoc

```diff
@@ -98,3 +96,29 @@ include::{sourcedir36}/impl/src/test/java/org/ehcache/docs/GettingStarted.java[t
 <5> Define the concurrency level of write-behind queue(s).
 This indicates how many writer threads work in parallel to update the underlying system of record asynchronously.
 <6> Enable the write coalescing behavior, which ensures that only one update per key per batch reaches the underlying system of record.
+
+== Cache-Through for Clustered Cache
+
+A clustered cache can also be configured with a `CacheLoaderWriter` for a cache-through use case.
+
+[source%nowrap,java,indent=0]
+----
+include::{sourcedir36}/clustered/client/src/test/java/org/ehcache/clustered/client/docs/ClusteredLoaderWriter.java[tag=clusteredWriteThroughCache]
+----
+
+<1> We create a sample `CacheLoaderWriter` with a known mapping for key `4L`.
```

Comment:
Can we drop the "we" from all these steps and rephrase these steps as generic instructions like "Create a sample..." or "Perform some cache operations" etc